dwitter/templates/dweet/dweet.html

```diff
@@ -39,6 +47,38 @@
           case "hideStats":
             setStatsVisibility(false);
             break;
+          case "record":
+            recording = true;
+            encoder = new GIF({ 
+              workers: 4, 
+              quality: 5, 
+              workerScript: "/static/libs/gif.worker.js",
+              width: c.width, 
+              height: c.width / 16 * 9,
+			  background: ""
+            });
+            FPS = 25;
+			gifCanvas.width = c.width;
+			gifCanvas.height = c.width / 16 * 9;
+			gifctx = gifCanvas.getContext("2d");
+            break;
+          case "stop":
```

Comment:
Rename to `stopGifRecord` as "stop" sounds like you're stopping the dweet, or anything else.